app/templates/course-admin/tester-version.hbs

```diff
@@ -0,0 +1,110 @@
+        <div>
+          <h3 class="flex items-start gap-x-3 text-lg font-bold text-gray-800 mb-2">
+            <p>{{@model.testerVersion.tagName}}</p>
+
+            {{#if @model.testerVersion.isActive}}
+              <Pill @color="green">Active</Pill>
+            {{else if @model.testerVersion.isLatest}}
+              <Pill @color="yellow">Latest</Pill>
+            {{/if}}
+          </h3>
+
+          <div class="prose prose-sm">
+            Created
+            {{date-from-now @model.testerVersion.createdAt}}
+          </div>
+
+          {{#if @model.testerVersion.isActive}}
+            <div class="flex items-center mt-6 gap-x-1 text-gray-600 text-sm">
+              <span>
+                Activated
+                {{date-from-now @model.testerVersion.lastActivatedAt}}
+              </span>
+
+              {{#if @model.testerVersion.activator}}
+                {{! @glint-expect-error user is not ts-ified yet !}}
+                <span>by {{@model.testerVersion.activator.username}}</span>
+              {{/if}}
+            </div>
+          {{else}}
+            <div class="flex items-center mt-6 gap-x-3">
+              <PrimaryButtonWithSpinner
+                @shouldShowSpinner={{this.isActivating}}
+                @size="small"
+                {{on "click" this.handleActivateButtonClick}}
+                data-test-activate-tester-version-button
+              >
+                Activate
+              </PrimaryButtonWithSpinner>
+
+              <TertiaryButtonWithSpinner
+                @shouldShowSpinner={{this.isDeprovisioningTestRunners}}
+                @size="small"
+                {{on "click" this.handleDeprovisionTestRunnersButtonClick}}
+                data-test-deprovision-test-runners-button
+              >
+                Deprovision Test Runners
+              </TertiaryButtonWithSpinner>
+            </div>
+          {{/if}}
+        </div>
+
+        <div class="flex flex-col items-end">
+          <a
+            href={{@model.testerVersion.viewReleaseLink}}
+            class="text-gray-400 flex items-center hover:text-gray-600"
+            target="_blank"
+            rel="noopener noreferrer"
+            data-test-view-release-link
+          >
+            {{svg-jar "github" class="inline-flex w-4 mr-1 fill-current"}}
+            <span class="text-xs underline font-medium">View release</span>
+          </a>
+        </div>
+      </div>
+
+      {{#if this.shouldShowDeprovisioningNotice}}
+        <Alert @type="success" class="relative mt-6">
+          <span class="leading-7 inline-block mr-10">
+            A job has been scheduled and test runners will be deprovisioned shortly.
+          </span>
+
+          <div class="absolute right-3 top-0 bottom-0 flex items-center justify-center">
+            <button
+              class="border border-green-300 p-1 rounded hover:bg-green-200"
+              type="button"
+              {{on "click" this.handleDismissButtonClick}}
```

Comment:
@libmartinito nice work! 

To keep things simple, let's remove the dismiss functionality. We'll add a `@dismissable=true` to `Alert` at some point, until then let's avoid the extra code as much as possible.